platforms/chibios/drivers/wear_leveling/wear_leveling_efl.c

```diff
@@ -135,7 +143,25 @@ bool backing_store_read(uint32_t address, backing_store_int_t *value) {
     uint32_t             offset = (base_offset + address);
     backing_store_int_t *loc    = (backing_store_int_t *)flashGetOffsetAddress(flash, offset);
     *value                      = ~(*loc);
+
+#ifdef EFL_CHECK_ECC
+    if (ecc_error) {
+        bs_dprintf("Failed to read from backing store, ECC error detected\n");
+        ecc_error = false;
+        return false;
+    }
+#endif // EFL_CHECK_ECC
```

Comment:
Should we also reset `ecc_error` before the read, so that any ECC error in a different place is not mistaken for an error in the backing store?

Reply:
Yep. Done.